tests/rules/no-collection-size-mischeck.test.ts

```diff
@@ -54,6 +54,16 @@ ruleTester.run('Collection sizes and array length comparisons should make sense'
           endLine: 1,
           column: 5,
           endColumn: 24,
+          suggestions: [
+            {
+              messageId: 'suggestFixedSizeCheck',
```

Comment:
can we assert concrete message here?

Reply:
Nope, TypeScript ESLint allows only messageId.